content/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow.md

```diff
@@ -70,9 +70,9 @@ These labels operate cumulatively, so a self-hosted runner’s labels must match
 
 When routing a job to a self-hosted runner, {% data variables.product.prodname_dotcom %} looks for a runner that matches the job's `runs-on` labels:
 
-1. {% data variables.product.prodname_dotcom %} first searches for a runner at the repository level, then at the organization level{% ifversion ghes or ghae %}, then at the enterprise level{% endif %}. 
-2. The job is then sent to the first matching runner that is online and idle.
-   - If all matching online runners are busy, the job will queue at the level with the highest number of matching online runners.
-   - If all matching runners are offline, the job will queue at the level with the highest number of matching offline runners.
-   - If there are no matching runners at any level, the job will fail.
+- {% data variables.product.prodname_dotcom %} first searches for an online and enabled runner at the repository level, then at the organization level{% ifversion ghes or ghae %}, then at the enterprise level{% endif %}.
+- If {% data variables.product.prodname_dotcom %} doesn't find an online and enabled runner at any level, the job is queued to all levels and waits for any runner from any level to come online and pickup the job.
```

Comment:
@TingluoHuang In the previous description, we said that `If there are no matching runners at any level, the job will fail.`. With this new behavior, if there are no runners configured at any level that match the specified labels for the job, will the job be queued and wait 24 hours before failing?

Reply:
The job will be queued and wait for 24 hours before failing. Within 24 hours, any label matched runner from any level (repo/org/enterprise) that comes online can pick up the job